test/Microsoft.DotNet.Web.Authentication.Tests/PersonalAccessTokenAuthenticationTests.cs

```diff
@@ -393,7 +393,7 @@ public string HashPassword(TestUser user, string password)
 
     public static string CalculateHash(TestUser user, string password)
     {
-        return $":HASH:{user.Name}:{password}"; // lgtm [cs/hardcoded-credentials] Part of unit tests
+        return $":HASH:{user.Name}:{password}"; // CodeQL [SM00423] Hardcoded as part of unit tests
```

Comment:
These aren't even hardcoded, this a string interpolation.

Reply:
I had to put 25 characters in there for a reason :D